src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/CollectionExerciseServiceImpl.java

```diff
@@ -679,22 +688,23 @@ public void transitionScheduleCollectionExerciseToReadyToReview(
     String searchStringJson = new JSONObject(searchStringMap).toString();
     Integer numberOfCollectionInstruments =
         collectionInstrumentSvcClient.countCollectionInstruments(searchStringJson);
-    boolean sampleLinksValid = validateSampleLinks(collexId);
+    boolean allSamplesActive = allSamplesActive(collexId);
     boolean shouldTransition =
-        sampleLinksValid
+        allSamplesActive
             && numberOfCollectionInstruments != null
             && numberOfCollectionInstruments > 0;
     log.info(
         "ready_for_review transition check:"
-            + "sampleLinksValid: {}, numberOfCollectionInstruments: {},"
+            + "allSamplesActive: {}, numberOfCollectionInstruments: {},"
             + " shouldTransition: {}",
-        sampleLinksValid,
+        allSamplesActive,
         numberOfCollectionInstruments,
         shouldTransition);
     if (shouldTransition) {
       transitionCollectionExercise(
           collectionExercise, CollectionExerciseDTO.CollectionExerciseEvent.CI_SAMPLE_ADDED);
     } else {
+      // TODO This isn't correct. If we cannot transition it doens't mean a sample has been deleted
```

Comment:
I'm not sure about this?

Reply:
I added this before creating a tech debt. https://trello.com/c/yFOxl7ki/162-collection-exercise-attempts-to-transition-based-on-cisampledeleted-when-a-sample-is-not-deleted. I'll remove